### PR TITLE
2022 redesign

### PR DESCRIPTION
**Pull request by aarongustafson**

### CODE REVIEW

**Comment by tropicadri** [comment]
I loooooove it!

**Comment by webmaxru** [comment]
Looks great!

File: templates/_includes/schedule.njk [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    </tr>
+    {% for session in day.rooms[0].sessions %} {% set even = false %} {% if loop.index % 2 == 0 %}
+    {% set even = true %} {% endif %}
+    <tr data-even="{{even}}" data-starts="{{session.startsAt}}" data-ends="{{session.endsAt}}">
+      <td class="schedule--time">
+        {% time session.startsAt, {hour: "numeric", minute: 'numeric'} %}
+      </td>
+      {% if session.isServiceSession == false %}
+      <td>
+        <a class="schedule--link" href="/sessions/{{session.title | slugify }}"
```
Comment (webmaxru) [inline templates/_includes/schedule.njk:47]:
```suggestion
        <a class="schedule--link" href="/sessions/{{ (session.title + session.speakers[0].name) | slugify }}"
```

This change should fix link to the session details from the mainpage

**Review by webmaxru — DISMISSED** [review]

**Comment by webmaxru** [comment]
Something is wrong with the timezone switch (or session timestamp in Sessionize export). This is what I see:
image
GMT+2 this is my timezone (Central Europe) and my workshop is not at 2AM :) It should be 11AM

**Comment by aarongustafson** [comment]
> GMT+2 this is my timezone (Central Europe) and my workshop is not at 2AM :) It should be 11AM

I need to look at @Snugug’s timezone code to see what’s what. I didn’t adjust from last year.

**Review by tropicadri — APPROVED** [review]

**Pull request merged**